make/k8s.sh

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env bash
+#
+# Helper script for kubernetes: deploy, destroy clusters and refresh tokens.
+
+export BASE_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+source ${BASE_DIR}/helpers.sh
+
+ACTION=${1}
+CLOUD_PROVIDER=${2}
+
+# This function provides help information for the script.
+function help() {
+  echo "Usage: $0 <command> [options]"
+  echo "Commands:"
+  echo "  help                               Display this help message."
+  echo "  deploy_k8s        <cloud_provider> Deploy Kubernetes on the specified cloud provider."
+  echo "  destroy_k8s       <cloud_provider> Destroy Kubernetes on the specified cloud provider."
+  echo "  refresh_token_k8s <cloud_provider> Refresh Kubernetes access token for the specified cloud provider."
+}
+
+# Check if the input parameters are correctly given.
+if [[ -z "${ACTION}" ]]; then
+  print_error "No command provided."
+  help
+  exit 1
+fi
+
+# Check if TFVARS_JSON is not defined and exit.
+if [ -z "${TFVARS_JSON}" ]; then
+  print_error "TFVARS_JSON is not defined."
+  exit 1
+fi
+
+# Check if the file pointed to by TFVARS_JSON does not exist and exit.
+if [ ! -f "${TFVARS_JSON}" ]; then
+  print_error "File '${TFVARS_JSON}' does not exist."
+  exit 1
+fi
+
+# Validate input values.
+if [[ "${ACTION}" == "deploy_k8s" || "${ACTION}" == "destroy_k8s" || "${ACTION}" == "refresh_token_k8s" ]]; then
```

Comment:
need to use arrays as previously mentioned

Reply:
ack, adopted